test/client_test.js

```diff
@@ -43,6 +43,7 @@ describe('Client', () => {
       it('returns SMART OAuth URIs', async function () {
         nock(this.baseUrl)
           .matchHeader('accept', 'application/json+fhir')
+          .matchHeader('Authorization', '')
```

Comment:
To reiterate: this line tests that requests for `/metadata` can work with an *empty* Authorization header...

Reply:
(and: it is empty, by default)